sdk/eventhub/Azure.Messaging.EventHubs.Processor/samples/AspNetHostedService/src/SampleProgram.cs

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Storage.Blobs;
+
+namespace Azure.Messaging.EventHubs.Processor.HostedService.Samples
+{
+    /// <summary>
+    /// The default entry point of the ASP.NET application.
+    /// </summary>
+    public class SampleProgram
+    {
+        /// <summary>
+        /// The default entry point method of the <see cref="SampleProgram" /> class.
+        /// </summary>
+        /// <param name="args"></param>
```

Comment:
We may want to remove the args parameter, since we're not actually accepting anything.   Otherwise, please add text to the doc comment.